test/onnx/test_utility_funs.py

```diff
@@ -206,6 +206,22 @@ def forward(self, A):
             assert node.kind() != "onnx::Transpose"
         assert len(list(graph.nodes())) == 1
 
+    def test_constant_fold_reshape(self):
```

Comment:
Can we add an end to end test for ORT that matches numbers for a model?

Reply:
Sure. I was considering the mask_rcnn test for that that I'm adding in a separate PR with other torchvision tests. I can include that test in this PR.